### PR TITLE
[13.0][IMP] website_sale_attribute_filter_category: Use sass instead of les…

### PR DESCRIPTION
**Pull request by mamcode**
…s for styles.

Sass seems to be the most used now for css styles. I didn't make any changes to the code just change the file extension and directory name to scss.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
It's not only the most used, but the one that avoids lessc dependency.

**Review by CarlosRoca13 — APPROVED** [review]
I test it and continue working! :+1: LGTM

File: website_sale_attribute_filter_category/views/assets.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
             <link
                 type="text/less"
```
Comment (Tardo) [inline website_sale_attribute_filter_category/views/assets.xml]:
```suggestion
                type="text/css"
```

**Review by Tardo — CHANGES_REQUESTED** [review]
Looks good! thanks, please update the mimetype too

**Comment by mamcode** [comment]
> Looks good! thanks, please update the mimetype too

Thanks :) Done.

**Review by HaraldPanten — APPROVED** [review]

**Review by Tardo — APPROVED** [review]

**Pull request merged**